stable/atlantis/README.md

```diff
@@ -42,6 +43,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `gitlab.token`                              | Personal access token for the Atlantis GitLab user.                                                                                                                                                                                                                                                       | n/a     |
 | `gitlab.secret`                             | Webhook secret for the Atlantis GitLab integration. All repositories in GitLab that are to be integrated with Atlantis must share the same value.                                                                                                                                                         | n/a     |
 | `gitlab.hostname`                           | Hostname of your GitLab Enterprise installation.                                                                                                                                                                                                                                                          | n/a     |
+| `secretWebhookName`                         | Name of the Kubernetes `Secret` where the webhook cretentials are stored (optional)                                                                                                                                                                                                                                                      | n/a     |
```

Comment:
```suggestion
| `vcsSecretsName`                         | Name of a pre-existing Kubernetes `Secret` containing `token` and `secret` keys set to your VCS provider's API token and webhook secret, respectively. Use this instead of `github.token`/`github.secret`, etc. (optional)                                                                                                                                                                                                                                                      | n/a     |
```

I know that the original secret is called `-webhook` but I find that pretty confusing since it contains the token (which isn't used when receiving webhooks but for API calls) *and* the webhook secret.

Could you call this `vcsSecretsName`?